src/server/models/types.ts

```diff
@@ -69,42 +69,40 @@ export enum UserRoles {
 
 type AsJsonObject<T> = T & JsonObject;
 
-export interface LawyerListItemCreateInput
-  extends PrismaClient.Prisma.ListItemCreateInput {
-  type: ServiceType.lawyers;
+type StringLike<T extends string> = T | `${T}` | string;
```

Comment:
What's this ``${T}``? Is it just converting the type to a string?

Reply:
Yes, converts to a string literal. https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-types 

I do this to circumvent a bunch of typing conflicts without casting all over the place, whilst still having a descriptive type. Ideally we don’t have string type at the end but I didn’t want to change even more code 😭.

Reply:
Ah I see, a `literla type` hack 😉